pepper-apis/src/main/java/org/broadinstitute/ddp/DataDonationPlatform.java

```diff
@@ -445,7 +445,7 @@ public void handle(Request request, Response response) throws Exception {
 
         // Routes calling DSM
         get(API.PARTICIPANT_STATUS, new GetDsmParticipantStatusRoute(new DsmClient(cfg)), responseSerializer);
-        get(API.STUDY_PASSWORD_REQUIREMENTS, new GetStudyPasswordRequirementsRoute(), responseSerializer);
+        get(API.STUDY_PASSWORD_POLICY, new GetStudyPasswordPolicyRoute(), responseSerializer);
```

Comment:
The password policy route calls DSM apis?

Reply:
No. Not sure why this line is here of all places, but I think the above line comment is just for the very next line after it.

Reply:
Let's see if we can either insert some more whitespace, singularize (or just remove the comment), or move the call elsewhere- the comment in that location, with that phrasing, is misleading.

Reply:
Agreed. Already moved it :)